plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -119,16 +122,20 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
 void chain_plugin::plugin_initialize(const variables_map& options) {
    ilog("initializing chain plugin");
 
+   my->block_interval_seconds = config::default_block_interval_seconds;
```

Comment:
This line and the next are redundant, since line 90 uses default_value, options will perform the same check.